### PR TITLE
Small edits

### PR DESCRIPTION
**Pull request by RichardLitt**
Take a look, @whyrusleeping.

### CODE REVIEW

File: src/21-pubsub/index.md [hunk 1]
```diff
@@ -23,20 +23,24 @@ template: tmpl/layouts/post.html
...
-To make ipfs fast for datacenters, local area networks, and large p2p
+To make IPFS fast for datacenters, local area networks, and large p2p
 applications, we need a fast publish subscribe system. [Publish
 Subscribe](https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern),
 called 'pubsub' for short, is a mechanism for 'publishers' to distribute
 messages quickly and directly to 'subscribers' that are interested in them.
-This is valuable because <somebody help why not use the phrase 'hella cool' in
-a blog post>.
+This is valuable because it decouples the publishers from the subscribers,
+so that data can still flow if a publisher goes down, or if there are no
```
Comment (whyrusleeping) [inline src/21-pubsub/index.md]: @jbenet what do you think?

File: src/21-pubsub/index.md [hunk 2]
```diff
@@ -71,10 +75,10 @@ To publish a message to the topic `foo`, open up another terminal and run:
 You should see "hello world" printed out in the first terminal.  You can also
 run the `pub` command on any other connected ipfs node and your node will
 receive the message.  Messages are routed through connected, subscribed peers.
-This means that if peers A,B, and C are all subscribed to foo, A is connected
+This means that if peers A,B, and C are all subscribed to `foo`, A is connected
 to B, and B is connected to C, but A is not directly connected to C, A will
-still receive messages that C published to "foo" through B. This can be very
-useful to route messages in networks with poor NAT traversal or otherwise
+still receive messages that C published to `foo` through B. This can be very
+useful for routing messages in networks with poor NAT traversal or otherwise
```
Comment (jbenet) [inline src/21-pubsub/index.md]: maybe just show a figure of the topology, too.

File: src/21-pubsub/index.md [hunk 3]
```diff
@@ -88,26 +92,26 @@ And to see all the topics you are currently subscribed to, run:
...
+that you can chat with Orbit on local area networks without a backbone internet
 connection, or in networks with spotty connectivity to the global internet.
 
 Aside from chat, there are many interesting possibilities. In the near future,
 IPNS records will be pushed over pubsub, allowing lightning fast updates of
-peers IPNS entries.  Peers could use pubsub to keep track of the head of some
+peers' IPNS entries.  Peers could use pubsub to keep track of the head of some
 [merkle-linked global log](https://en.wikipedia.org/wiki/Blockchain_(database)).
-Pubsub could even be used to [(inefficiently) route IP packets](https://ipfs.io/ipfs/QmUVtpaAUK7rNimRstqhk98PTMEu6PjBfmHBanz3LxRU5p/main.go) between peers,
+Pubsub could even be used to [(inefficiently) route IP packets](https://ipfs.io/ipfs/QmUVtpaAUK7rNimRstqhk98PTMEu6PjBfmHBanz3LxRU5p/main.go) between peers.
```
Comment (jbenet) [inline src/21-pubsub/index.md]: if you reaaally want to keep this, then mention IP Multicast.

File: src/21-pubsub/index.md [hunk 4]
```diff
@@ -88,26 +92,26 @@ And to see all the topics you are currently subscribed to, run:
...
 Any existing web protocol could be tunneled over pubsub with relative ease.
 
 
-## Whats next?
+## What's next?
 The next steps for pubsub have to do with authentication. Currently, any peer
 can publish to any pubsub topic. In the near future, we are going to implement
 an authenticated mode of pubsub that allows the selection of peers who are
-authorized to publish messages to a topic. This would allow certain usecases to
+authorized to publish messages to a topic. This would allow certain use-cases to
```
Comment (jbenet) [inline src/21-pubsub/index.md]: > an authenticated mode for pubsub topics, where only certain authorized peers -- those given a cryptographic key or _capability_ --  can publish messages. We are still working out the sharing and capability granting model.

**Comment by jbenet** [comment]
@whyrusleeping i reviewed

**Pull request merged**